.github/workflows/ci.yml

```diff
@@ -68,6 +68,26 @@ jobs:
           name: BeamAdapter_${{ steps.sofa.outputs.run_branch }}_for-SOFA-${{ steps.sofa.outputs.sofa_version }}_${{ runner.os }}
           path: ${{ env.WORKSPACE_ARTIFACT_PATH }}
       
+      - name: Set env vars for tests
+        shell: bash
+        run: |
+          # Set env vars for tests
+          if [[ "$RUNNER_OS" == "Windows" ]]; then
+            echo "$WORKSPACE_ARTIFACT_PATH/lib" >> $GITHUB_PATH
```

Comment:
if I understood well, it will print the (windows-style) current path into github_path, aka /blabla/artefact/BeamAdapter/lib.
so it seems equivalent to echo ` "$WORKSPACE_ARTIFACT_PATH/lib" >> $GITHUB_PATH` 
(maybe some win32 subtilities i am not aware of)